Getting-Started/Design/Rendering-Media/index.md

```diff
@@ -1,40 +1,71 @@
 # Rendering media
 
-_Templates can access items in the Media library, to assist in displaying rich content like galleries_
+_Templates can access items in the [Media library](../../Data/Creating-Media/index.md), to assist in displaying rich content like galleries_
 
-## Rendering a single media item
-A media node is not just a file, but a collection of fields, such width, height and the path to the stored file.
+In the following examples we will be looking at rendering out an Image, however the core principles do apply to all media items but property aliases may differ.
 
+## Rendering a media item
+A media node is not just a file, but like content, it is a collection of fields, such width, height and the path to the stored file. The benefit of this is that accessing media is very similar to accessing a content node.
+
+### Example 1: Accessing an image media item based on its ID
 A standard image in the media library is based on the Mediatype `image` which provides a number of standard values - if you want to add more, simply edit the media type under **settings**.
 
+_Assumption: We are going to assume that our media item has an ID of **1234**, and that we are **not using Models Builder**_
+
     @{
-        var mediaItem = Umbraco.Media(1234);
-        var height = mediaItem.UmbracoHeight;
-        var file = mediaItem.UmbracoFile;
+        var mediaItem = Umbraco.TypedMedia(1234);
+        var height = mediaItem.GetPropertyValue<string>("umbracoHeight");
+
+        //There are two ways to get your media item url, this is one way.
+        var file = mediaItem.GetPropertyValue<string>("umbracoFile");
+
+        //This is the other
+        var url = mediaItem.Url
     }
 
     <img src="@file" height="@height" />
```

Comment:
Perhaps rendering the name as alt text might be a better example than height?

Reply:
Yeah no problem, I'll get on that. I only did the height as that was what the original one did :-)